tagstudio/src/core/library.py

```diff
@@ -735,17 +740,20 @@ def to_json(self):
 
         file_to_save: JsonLibary = {
             "ts-version": VERSION,
-            "ignored_extensions": [],
+            "ext_list": [],
+            "is_exclude_list": True,
             "tags": [],
             "collations": [],
             "fields": [],
             "macros": [],
             "entries": [],
+            "ignored_extensions": [],
         }
 
         print("[LIBRARY] Formatting Tags to JSON...")
 
-        file_to_save["ignored_extensions"] = [i for i in self.ignored_extensions if i]
+        file_to_save["ext_list"] = [i for i in self.ext_list if i]
+        file_to_save["is_exclude_list"] = self.is_exclude_list
```

Comment:
you can do these assignment above on lines 743-744 instead of having there the dummy values